content/02.introduction.md

```diff
@@ -121,7 +121,24 @@ The mechanism behind these insults has been suggested to be related to inflammat
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
+#### Subpopulations of Special Concern  
+
+The prevalence of different clinical features of COVID-19 infection differ in different areas of the world, presumably due to differential reporting and/or different levels of pre-existing risk factors [@doi:10.1007/s00592-020-01546-0].
+In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
+African Americans have also been reported to have disproportionate risk for kidney complications for COVID-19 [@doi:10.1016/j.kint.2020.05.006].
+The racial disparity may be due to a number of factors, including different vulnerability due to higher levels of a pre-existing conditions such as diabetes, to greater economic difficulties or more hazardous work, or to less access to or trust in the health care system [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006].
+Less health care access could cause infections to be less likely to be diagnosed unless or until they are very severe. 
+However, more research is needed into studying and and remediating these disparities. 
+Long-term damage caused by chronic stress related to aversive social experiences [@doi:10.1101/2020.05.10.20097253] leading to cardiovascular risk [@doi:10.1007/s12170-013-0338-5] might be relevant but has not yet been researched in the specific case of COVID-19 disparities.
```

Comment:
the first reference you cite here should be placed elsewhere in the sentence because it doesn't really discuss what is immediately preceding it